compiler/src/dotty/tools/dotc/cc/CaptureOps.scala

```diff
@@ -198,7 +198,7 @@ extension (tp: Type)
       || tp.isRootCapability
       ) && !tp.symbol.isOneOf(UnstableValueFlags)
     case tp: TypeRef =>
-      tp.symbol.isAbstractOrParamType && tp.derivesFrom(defn.Caps_CapSet)
```

Comment:
@odersky Do you remember why we require the arguments of `CapsOf` to be abstract before? I think any type derived from CapSet should work.